### ISSUE TITLE
Something wrong with reproduction

### ISSUE THREAD

**Issue by renjiawei1992** [salesforce/rng-kbqa#22]
I am very interested in your project and thank you for your code. but when I use Reproducing the Results on GrailQA/Step by Step Instructions/(ii) Disambiguating Entities (Entity Linking), it will stuck at entity disambiguating. I do not know what is going wrong, which seems something goes wrong when running the code: "logits = model(**inputs)[1]", can you help me?

![1386193464](https://user-images.githubusercontent.com/42289220/235345768-22266dd0-915e-4bea-8509-e332890d6a0e.jpg)

**Comment by xiye17** [comment]
I can't tell what's happening here. Seems model in ln 275 returns `None`. I would suggest checking transformer/pytorch versions and zoom in model files to see why it returns `None`.

**Comment by Enqurance** [comment]
Same issue
